ansible/roles/openshift-monitoring-config/templates/cluster-monitoring-config.yml.j2

```diff
@@ -1,37 +1,42 @@
----
 apiVersion: v1
 kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
 data:
-  config.yaml: |
-    prometheusOperator:
-      baseImage: quay.io/coreos/prometheus-operator
-      prometheusConfigReloaderBaseImage: quay.io/coreos/prometheus-config-reloader
-      configReloaderBaseImage: quay.io/coreos/configmap-reload
+  config.yaml: |+
     prometheusK8s:
+      nodeSelector:
+        {{ openshift_prometheus_node_selector }}
       retention: {{ openshift_prometheus_retention_period }}
-      baseImage: openshift/prometheus
       volumeClaimTemplate:
+        metadata:
+          name: localpvc
```

Comment:
Is the metadata field required?  I don't believe we needed it in the past but maybe it's needed for something else?

Reply:
This is taken from a working cluster-monitoring-configmap I've been using on the large cluster. I don't see it harming us anyway.